docs/resources/host.md

```diff
@@ -54,8 +54,7 @@ BIOS, HBA, SSD, HDD, etc. of the host must match the VMware Hardware Compatibili
 
 ### Read-Only
 
-- `host_id` (String) UUID of the host. Known after commissioning.
```

Comment:
This is the right change, but just note that these are breaking changes and we need to put in a policy to avoid such changes once we reach 1.0 release of the provider.

Please add this breaking change to the release notes (I am guessing this was part of 0.3.0 release).

Reply:
Completely agree!

Reply:
Yeah, that is why I decided to do it now, while the userbase is still low and there won't be too much of a disruption.